exercises/practice/space-age/.meta/config.json

```
@@ -1,6 +1,12 @@
 {
   "blurb": "Given an age in seconds, calculate how old someone is in terms of a given planet's solar years.",
-  "authors": [],
+  "authors": [
+    "devkabiir"
+  ],
+  "contributors": [
+    "ErikSchierboom",
+    "Stargator"
+  ],
```

```suggestion
  "contributors": [
  ],
```

---

I suggest adding my name due to [an update for values in some test cases](https://github.com/exercism/dart/commit/0567ef74d2ab08134e00fb1be42c42e2d7e8117c#diff-e4887664f150fb9cb9cc6ebbbb8ab74828be82c9a31661dffe32efbc5dd046dc).

```suggestion
  "contributors": [
    "Stargator"
  ],
```